apps/manager_app/device_authentication.c

```diff
@@ -172,16 +168,26 @@ static pb_size_t get_request_type(
   return request->which_request;
 }
 
-static manager_auth_device_response_t send_flow_complete(void) {
+static void send_auth_device_response(manager_auth_device_response_t *resp) {
+  manager_result_t result =
+      get_manager_result_template(MANAGER_RESULT_AUTH_DEVICE_TAG);
+  memcpy(&(result.auth_card), resp, sizeof(manager_auth_device_response_t));
```

Comment:
confusing as we are in device auth and copying into `result.auth_card`.

Reply:
Fixed! Good catch